.github/workflows/publish-to-third-party.yml

```diff
@@ -1,50 +1,76 @@
 name: Auto publish release to external stores
 
 on:
-  release: 
-    # Triggers when a release was published, or a pre-release was changed to a release.
-    # https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=prereleased#release
-    # https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=released#release
-    # https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=edited#release
-    types: 
-      - released
-      - prereleased
-      - edited
-
-run-name: Auto publishing ${{ github.event.release.tag_name }} to external stores
+  workflow_call:
+    inputs:
+      tag:
+        description: "The tag of the release to publish"
+        type: string
+        required: true
+    secrets:
+      BOT_TOKEN:
+        required: true
+        description: "The token used to authenticate to the Github API"
+  
+  workflow_dispatch:
+      inputs:
+        tag:
+          description: "The tag of the release to publish"
+          type: string
+          required: true
+
+
+run-name: Auto publishing ${{ inputs.tag }} to external stores
 
 jobs:
+  get-tag-info: 
+    runs-on: ubuntu-latest
+    name: Get tag info 🏷️
+
+    outputs:
+      tag: ${{ steps.release_data.outputs.tag_name }}
+      prerelease: ${{ steps.release_data.outputs.prerelease }}
+
+    steps:
+      - name: git-get-release-action
+        uses: cardinalby/git-get-release-action@1.2.4
+        id: release_data
+        with:
+          tag: 1.9.3
```

Comment:
As this workflow uses a `tag` parameter, it would be great to use it here to have this file always working :)

Reply:
Oops I forgot to change this, fixed :)